rclpy/test/test_rate.py

```diff
@@ -101,14 +101,16 @@ def test_rate_invalid_period(self):
 
     def test_destroy(self):
         rate = self.node.create_rate(FREQ)
-        self.node.destroy_rate(rate)
+        success = self.node.destroy_rate(rate)
+        assert success is True, "destroy_rate() returned %r instead of True" % success
```

Comment:
These asserts are a bit too verbose in my opinion. `assert self.node.destroy_rate(rate)` is fine, especially with how verbose the output is from pytest when a test assertion fails.

Reply:
Made it so.